### ISSUE TITLE
A task for identify N words in the audio

### ISSUE THREAD

**Issue by ftyers** [d33tah/commonvoice-languagelearning#54]
The task:
* Play an audio
* Get presented with e.g. 6 words (or Chinese characters)
   * 3 are in the audio, 3 are not (distractors)
* You have to click on the ones that are there, without the ones that aren't.

Thanks to @JacobSchmitt for the idea!

**Comment by d33tah** [comment]
Looks like a variant of #56 to me?

**Comment by ftyers** [comment]
Yeah, I suppose it's kind of the opposite of that, so could be done using the same code modulo a boolean.

**Comment by JacobSchmitt** [comment]
option for number of unknown words/blanks per card (2 instead of 1)
      basically increase difficulty until you're writing whole sentences

**Comment by JacobSchmitt** [comment]
option to click on words in the sentence you find hard and then automatically generate a flashcard to review later that also provides the context (audio and text)

**Comment by ftyers** [comment]
Here is a first pass in de5b12f:
![imatge](https://user-images.githubusercontent.com/449545/110193671-36072c80-7e2d-11eb-9c78-ca4dc3eeddf0.png)

**Comment by ftyers** [comment]
Not clear what the rules should be, e.g. 
- :star: get 3/3
- ✔ get 2/3
- ❀ get 1/3
- ✘ get 0/3

Should it let you only select a total of three?

**Comment by ftyers** [comment]
I think the way to go for the first version is basically: you get three clicks. The system tracks clicks and correct. If `nclicks == correct_clicks` then you get a tick, otherwise nothing.

**Comment by ftyers** [comment]
This now works as of 8f099cd.
